pubsub/in-memory/in-memory.go

```diff
@@ -0,0 +1,44 @@
+package inmemory
+
+import (
+	"context"
+
+	"github.com/asaskevich/EventBus"
+	"github.com/dapr/components-contrib/pubsub"
+	"github.com/dapr/kit/logger"
+)
+
+type InMemoryBus struct {
+	bus EventBus.Bus
+	ctx context.Context
+}
+
+func New(logger logger.Logger) pubsub.PubSub {
+	return &InMemoryBus{}
+}
+
+func (a *InMemoryBus) Close() error {
+	return nil
+}
+
+func (a *InMemoryBus) Features() []pubsub.Feature {
+	return nil
+}
+
+func (a *InMemoryBus) Init(metadata pubsub.Metadata) error {
+	a.bus = EventBus.New()
+	a.ctx = context.Background()
+	return nil
+}
+
+func (a *InMemoryBus) Publish(req *pubsub.PublishRequest) error {
+	a.bus.Publish(req.Topic, a.ctx, req.Data)
+	return nil
+}
+
+func (a *InMemoryBus) Subscribe(req pubsub.SubscribeRequest, handler pubsub.Handler) error {
+	a.bus.SubscribeAsync(req.Topic, func(ctx context.Context, data []byte) {
```

Comment:
`SubscribeAsync` does return an error, so it's probably useful to check for that and handle it (even if you expect as an invariant that the call should never error today)

Reply:
I'm not really sure I need the `Async` version of the method since both work fine when I load the component into dapr, but the other variant is blocking the tests and I couldn't figure it out using goroutines.